Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM wordpress:5.3
```

Comment:
I think we can use wordpress:latest here to not be bound by a specific version and to make it future proof without having to update the file every time a new version of WP is released.

Reply:
I wasn't sure if `wordpress:latest` always pull the latest stable version or if it could be the nightly build which is why I decided to specify the version. I see, however, that at least for now, when using `wordpress:latest`, 5.3 is being installed on the site. I also see that we use `wordpress:latest` for running unit tests for example. I think it is OK to change it to `wordpress:latest` which is what I did.